omnibus/config/software/datadog-process-agent.rb

```diff
@@ -4,7 +4,6 @@
 # Copyright 2018 Datadog, Inc.
 
 name "datadog-process-agent"
-always_build true
```

Comment:
I wouldn't remove this for non-development builds: unless the `process_agent_version ` changes between 2 builds, I think omnibus will regard the previously downloaded artifact as being up-to-date. So for example, for consecutive builds with a `process_agent_version` set to `master`, as long as the omnibus cache is present I'd expect omnibus to never re-download the process-agent binary even if the binary for `master` on S3 has changed. When we re-enable the omnibus cache on our build pipeline this means that the process-agent won't be updated in non-tagged builds the way it should be, which is an unexpected behavior.

So, overall, I'd advise setting this to `false` only if the user really means that they don't care which version of the process-agent is included in the package.

Reply:
The rationale here is to avoid this recipe invalidating the build cache.
Omnibus cache invalidation is pretty naive: if one item has been rebuilt, assume all the following items need to be rebuilt also.

I'll revert this, and investigate an optional "quick-build" switch to skip the systematic rebuild